knowledge_base/Environment-Variables.md

```diff
@@ -0,0 +1,394 @@
+
+## DD_DYNO_HOST
+
+Boolean set in our Heroku `commonwealthapp` pipeline, in order to scope the buildpack to a dyno. See [DataDog.md](../knowledge_base/Datadog.md) for more info.
+
+## DD_ENABLE_HEROKU_POSTGRES
+
+Boolean set in our Heroku `commonwealthapp` pipeline to enable PSQL analytics through DataDog. See [DataDog.md](../knowledge_base/Datadog.md) for more info.
+
+## DD_ENABLE_HEROKU_REDIS
+
+Boolean set in our Heroku `commonwealthapp` pipeline to enable Redis analytics through DataDog. See [DataDog.md](../knowledge_base/Datadog.md) for more info.
+
+## DD_DISABLE_HOST_METRICS
+
+Boolean in our Heroku `commonwealthapp` pipeline to disable metrics that are not presently useful to us. As of 240101, should be disabled (i.e. `true`). See [DataDog.md](../knowledge_base/Datadog.md) for more info.
+
+## DD_LOG_LEVEL
+
+As of 231212, set to `WARN` in our Heroku `commonwealthapp` pipeline. DataDog configuration token determining severity level logged (e.g. ). May be set to `OFF`, `CRITICAL`, `ERROR`, `WARN`, `INFO`, `DEBUG`, or `TRACE`. See [DataDog.md](../knowledge_base/Datadog.md) for more info.
+
+## DD_SITE
+
+DataDog configuration token in our Heroku pipeline, specifying our DataDog site region. As of 231212, that region is `us5.datadoghq.com`. For more documentation, see [DataDog docs](https://docs.datadoghq.com/getting_started/site/) and [DataDog.md](../knowledge_base/Datadog.md).
+
+## DISABLE_CACHE
+
+If `true`, disables Redis caching middleware.
+
+## DISCORD_BOT_TOKEN
+
+This value should mirror the value of `DISCORD_TOKEN` in the Discobot .env file.
+
+## DISCORD_BOT_URL
+
+<!-- This token appears to be a holdover from the first iteration of Discobot, but is still (likely erroneously) referenced in the Snapshot Consumer. Flagged for removal. -->
+
+## DISCORD_CLIENT_ID
+
+For local testing, we use the staging Discord app/bot. The client ID can therefore be found on the [developer dashboard](https://discord.com/developers/applications/1027997517964644453/oauth2/general) or by contacting Jake or Timothee.
+
+## DISCORD_WEBHOOK_URL_DEV
+
+Connects to the #webhook-testing Discord channel on the Commond Protocol Discord server. Required to use the emit-webhook script to send Discord webhooks. More info at [Webhooks.md](./Webhooks.md).
+
+## DL_BUILD
+
+Boolean which ensures that Heroku only builds the Discobot package and related code when deploying.
+
+## ENFORCE_SESSION_KEYS
+
+Boolean feature flag for server-side enforcement of Canvas session keys; by default, `false`.
+
+Owner: Raymond Zhong.
+
+## ETH_ALCHEMY_API_KEY
+
+Used in our CI/CD and stored in GitHub repo secrets. If set, the `load-db` package script will replace production Alchemy URLs with their locally supported variants. Only needed if doing work that involves querying Ethereum.
+
+Owner: Kurtis Assad.
+
+## ETH_RPC
+
+<!-- In need of documentation; if you have information about this token, please contribute. -->
```

Comment:
Owned by @ianrowan -- injects test metamask wallet info, in my understanding.

Reply:
Reached out & resolved!